src/main/java/org/jenkinsci/plugins/workflow/multibranch/WorkflowMultiBranchProject.java

```diff
@@ -133,7 +134,8 @@ private BranchJobProperty reconstructBranchJobProperty(@NonNull WorkflowJob job,
                 strategy != null ? strategy.getPropertiesFor(head) : Collections.emptyList()));
     }
 
-    @Extension public static class DescriptorImpl extends MultiBranchProjectDescriptor implements IconSpec {
+    @Extension(ordinal = 1000) @Symbol({"multibranch"})
```

Comment:
Why the `ordinal`? Is that somehow related to adding a symbol?

Reply:
you are right, got confused.